src/transactions/feed/TransactionFeedV2.tsx

```diff
@@ -203,58 +218,16 @@ function mergeStandByTransactionsInRange({
     return inRange || newTransaction || veryOldTransaction
   })
   const deduplicatedTransactions = deduplicateTransactions([...transactions, ...standByInRange])
-  const transactionsFromAllowedNetworks = deduplicatedTransactions.filter((tx) =>
-    allowedNetworks.includes(tx.networkId)
-  )
```

Comment:
Moved this to be applied at the end of the processing flow, within `sections` const. In this case, we can ensure that if somehow some async changes from mutlichain arrive and change `showTransfers` - it will instantly be reflected with recalculation of `sections`